src/main/java/uk/gov/hmcts/ccd/data/casedetails/query/CaseDetailsQueryBuilder.java

```diff
@@ -121,11 +121,13 @@ public CaseDetailsQueryBuilder whereMetadata(MetaData metadata) {
         return this;
     }
 
-    public CaseDetailsQueryBuilder orderByCreatedDate(String sortDirection) {
+    public CaseDetailsQueryBuilder orderBy(MetaData metadata) {
+        String sortDirection = metadata.getSortDirection().orElse("asc");
```

Comment:
in theory this changes a bit the behaviour of the current standard API. Because before we always got sort by Created Date. Now this could be changed on the standard API for any case type. Not a big deal though

Reply:
It does change the behaviour of this method, although that sort field is not passed by the standard API, so it will always be null and use Created date to sort. The aggregated API explicitly sets the sort field for DIVORCE. So this change is safe.